scss/detail.scss

```diff
@@ -0,0 +1,56 @@
+@import 'variables';
```

Comment:
Import this in `application.scss` so you won't need to import it on every SASS file 😄

Reply:
If i import it on 'application.scss' and i delete the one in the 'details.scss' then it loses the references to the variables. Maybe im missing another step?

Reply:
Maybe for the linter it will happen but when compiled, it will work just fine